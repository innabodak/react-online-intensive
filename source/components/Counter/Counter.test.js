//Core
import React from 'react';
import Counter from './';
import renderer from 'react-test-renderer';

const rendererTree = renderer.create(<Counter count = { 3 } />).toJSON();

test('counter component should correspont to its shapshot counterpart', () => {
    expect(rendererTree).toMatchSnapshot();
});
