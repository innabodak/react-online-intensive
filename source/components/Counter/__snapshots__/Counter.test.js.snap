// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`counter component should correspont to its shapshot counterpart 1`] = `
<section
  className="counter"
>
  Post count: 
  3
</section>
`;
